docs/pages/badge.js

```diff
@@ -25,51 +24,255 @@ export default function BadgePage({ generatedDocGen }: {| generatedDocGen: DocGe
+        </Table.Cell>
+      </Table.Row>
+      <Table.Row>
+        <Table.Cell>
+          <Text>Error</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="300">Ads & Campaigns <Badge text="Failed" type="error"/></Text>
+        </Table.Cell>
+      </Table.Row>
+      <Table.Row>
+        <Table.Cell>
+          <Text>Neutral</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="300">Ads & Campaigns <Badge text="Not started" type="neutral"/></Text>
+        </Table.Cell>
+      </Table.Row>
+    </Table.Body>
+  </Table>
 `}
-      />
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Over media"
+          description={`
+          Badge may be used over media using two wash styles.
 
-      <Example
-        description={`
-    Components like [Module](/module) and [Dropdown](/dropdown) support Badges within the component.`}
-        name="Example: within other components"
-        defaultCode={`
+          1. **Over media - Light wash**
+          The light wash badge should be used over media that is dark or utilizes a dark gradient overlay.
+
+          2. **Over media - Dark wash**
+          The dark wash badge should be used over media that is light or utilizes a light gradient overlay.
+`}
+        >
+          <MainSection.Card
+            defaultCode={`
+<Flex gap={4}>
+  <Mask height={150} width={280} rounding={2}>
+    <Image alt="Botanical art in coral and green" fit="cover" src="https://i.ibb.co/cbjgZft/img-door.jpg" naturalWidth={1} naturalHeight={1}/>
+    <Box position="absolute" top>
+      <Box padding={4}>
+        <Badge text="Light wash" type="lightWash"/>
+      </Box>
+    </Box>
+  </Mask>
+  <Mask height={150} width={280} rounding={2}>
+    <Image alt="Botanical art in coral and green" fit="cover" src="https://i.ibb.co/7bQQYkX/stock2.jpg" naturalWidth={1} naturalHeight={1}/>
+    <Box position="absolute" top>
+      <Box padding={4}>
+        <Badge text="Dark wash" type="darkWash"/>
+      </Box>
+    </Box>
+  </Mask>
+</Flex>
+          `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Positioning"
+          description="By default, Badge is rendered inline within the parent element. However, the `position` prop can be used to adjust the alignment. Badges should align to the top of large accompanying text."
+        >
+          <MainSection.Card
+            defaultCode={`
+<Flex gap={4} direction="column">
+  <Text size="300">Ads & Campaigns <Badge text="New" /></Text>
+  <Text size="600">Ads & Campaigns <Badge text="Beta" position="top"/></Text>
+</Flex>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Within other components"
+          description="Components like [Module](/module) and [Dropdown](/dropdown) support Badges within the component."
```

Comment:
This should be more explicit — something about how those components have Badge built-in, so using this component isn't necessary